README.md

```
@@ -1 +1,179 @@
+
+- name: Run Spring Boot application
+  run: mvn spring-boot:run &
+  # Run Spring Boot in the background
+
+- name: Wait for 1 minute (simulate a short-running process)
+  run: sleep 60  # Sleep for 1 minute
+
+- name: Check application health
+  run: |
+    echo "Checking application health..."
+    curl -s --head http://localhost:8080/actuator/health | head -n 10
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)